tools/lxdclient/client.go

```diff
@@ -261,6 +272,17 @@ func verifyDefaultProfileBridgeConfig(client *lxd.Client) (string, error) {
 	// with it, so let's just use whatever they set up.
 	eth0, ok := config.Devices[configEth0]
 	if !ok {
+		/* on lxd >= 2.3, there is nothing in the default profile
+		 * w.r.t. eth0, because there is no lxdbr0 by default. Let's
+		 * handle this case and configure one now.
+		 */
+		if networkAPISupported {
+			if err := CreateDefaultBridgeInDefaultProfile(client); err != nil {
+				return "", errors.Errorf("couldn't create default bridge %v", err)
+			}
+
+			return network.DefaultLXDBridge, nil
+		}
 		return "", errors.Errorf("unexpected LXD %q profile config without eth0: %+v", defaultProfileName, config)
```

Comment:
In fact I don't think it applies here, because there's no error here to annotate.